src/crypto/tls/common.go

```diff
@@ -191,6 +191,8 @@ var supportedSignatureAlgorithms = []SignatureScheme{
 	ECDSAWithSHA1,
 }
 
+var supportedSignatureAlgorithmsDC = supportedSignatureAlgorithms
```

Comment:
This one here, I'm not really sure. Let me think about it.

Reply:
It was just something to get stuff working here — I'll definitely defer to your implementation when we rebase.